src/main/java/io/skai/okta/internshipstreamingserviceparserxml/SpringConfig.java

```diff
@@ -0,0 +1,9 @@
+package io.skai.okta.internshipstreamingserviceparserxml;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@PropertySource("classpath:application.properties")
```

Comment:
this is redundant, application.properties is reserved by spring boot and will be processed automatically